demisto_sdk/commands/validate/tests/RM_validators_test.py

```diff
@@ -1126,3 +1128,50 @@ def test_invalid_short_file():
         not_to_short_readme_validator.obtain_invalid_content_items([test_pack])
     )
     assert result[0].message == short_readme_error
+
+
+def test_ImagePathIntegrationValidator_content_assets():
+    """
+    Given
+    content_items.
+    - Pack with:
+        1. invalid readme that contains absolute path. For example:
+            - https://www.example.com/content-assets/example_image.jpg
+        2. invalid description contains relative path that saved not under doc_files. For example:
```

Comment:
```suggestion
        2. An invalid description contains a relative path not saved under doc_files. For example:
```
Please edit other descriptions as suggested here.